tests/kernel/boot_ltp.pm

```diff
@@ -80,4 +98,11 @@ sub test_flags {
 
 See run_ltp.pm.
 
+=head2 LTP_SELINUX
```

Comment:
If new variables defined, should we add description into the variables.md instead of here?